specification/authorization/resource-manager/Microsoft.Authorization/2015-07-01/authorization.json

```diff
@@ -1038,82 +625,6 @@
       },
       "description": "Provider operations metadata list"
     },
-    "RoleAssignmentPropertiesWithScope": {
```

Comment:
>RoleAssignmentPropertiesWithScope [](start = 5, length = 33)

Removing operations from an existing API version will cause breaking changes on generated code.
It is better to create a file with the operations that will be modified in the next API version, so the tag for 2015 version calls this two files (operations that wont change, and operations that will change), and the tag for 2017 version calls the other two files (the one with the unmodified operations and the one with the modified operations).

Reply:
Since we would need the 2017 tag's[as per above] generated files for sdk-for-net.How would this work if we have two tags?
which generated files will be used by sdk-for net then?

Reply:
In the `Basic Information` section you can specify which tag is the default one to generate.

Reply:
thanks,will update the pr as per the comments then